modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -160,6 +160,9 @@ GLOBAL_LIST_EMPTY(valid_cryopods)
 	/// Has the occupant been tucked in?
 	var/tucked = FALSE
 
+	/// What was the ckey of the client that entered the cryopod?
+	var/stored_ckey = FALSE
```

Comment:
just have this be an empty string or `null` instead of a bool

Reply:
done